args/args.go

```diff
@@ -103,6 +109,14 @@ func (g *GeneratorArgs) LoadGoBoilerplate() ([]byte, error) {
 		return nil, err
 	}
 	b = bytes.Replace(b, []byte("YEAR"), []byte(strconv.Itoa(time.Now().Year())), -1)
+
+	if g.GeneratedByCommentTemplate != "" {
+		generatorPath := strings.Split(os.Args[0], "/")
+		generatorName := generatorPath[len(generatorPath)-1]
+		generatedByComment = strings.Replace(g.GeneratedByCommentTemplate, "GENERATOR_NAME", generatorName)
+		s := fmt.Sprintf("\n%s\n\n", generatedByComment)
```

Comment:
The first newline is not necessary if the file is empty (which it is e.g. in OpenShift)

Reply:
I think it could be removed entirely, since the licences usually end with an extra \n

Reply:
Also it would be hard to tell if the the newline should be printed if there is ```// +build !ignore_autogenerated``` above the generated by comment?